graphite/src/main/scala/fr/davit/akka/http/metrics/graphite/CarbonClient.scala

```diff
@@ -20,7 +19,7 @@ object CarbonClient {
 
 class CarbonClient(host: String, port: Int)(implicit system: ActorSystem) extends AutoCloseable {
 
-  implicit private lazy val materializer: Materializer = ActorMaterializer()
+  implicit private lazy val materializer: Materializer = Materializer(system)
```

Comment:
If I check the [akka 2.6 migration guide](https://doc.akka.io/docs/akka/current/project/migration-guide-2.5.x-2.6.x.html#akka-stream-changes), I'm not sure we need to declare a materializer at all now

Reply:
Yes, they now have implicit def which converts actor system to mat
I have corrected it.